### PR TITLE
feat: add opengraph to views

### LINKED ISSUE #135: Improve SEO and OpenGraph Info
# Description

image

When linking the site, it just copies the `<title>` and a one-line description. It would be nice to have it show the logo, a better description, and when on a reference/compare page, show the two languages.

# Requirements

* Add OpenGraph (i.e. `og:title` type tags) to make it appear when linked to give a better description of the page.
* Info should be on the home page, compare, and research pages. (Look through the templates to see what's there and where it could be added.)
* Add a unit test for this as well

# Additional Notes

There are plugins for the browser like [Open Graph Preview](https://addons.mozilla.org/en-US/firefox/addon/open-graph-preview-and-debug/) for Firefox and probably similar for Chrome. They will show you what the page's info looks like, and how it might look when pasted on Twitter/Facebook/Slack/Discord/etc.

### PR DESCRIPTION
**Pull request by lukeberry99**
## What GitHub issue does this PR apply to?

Resolves #135

## What changed and why?

Added opengraph for improved SEO

## Checklist

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work
- [ ] (If editing Django) I have added or edited any appropriate unit tests for my changes

## Any additional comments or things to be aware of while reviewing?

No unit tests have been added because I can't find a way to test that the meta tags have been rendered inside the view with the current testing setup, I'm happy to create a new issue for adding additional testing functionality and to start working on implementing that.

### CODE REVIEW

File: web/views.py [hunk 1]
```diff
@@ -185,7 +187,8 @@ def compare(request):
         "lang1_friendlyname": lang1.friendly_name,
         "lang2_friendlyname": lang2.friendly_name,
         "categories": both_categories,
-        "concepts": both_concepts
+        "concepts": both_concepts,
+        "description": "Code Thesaurus: Comparing " + lang1.friendly_name + " " + lang2.friendly_name
```
Comment (geekygirlsarah) [inline web/views.py]: Could you add `and` between the languages? Otherwise it'll be like "Comparing Ruby Python" which doesn't make sense.

**Comment by geekygirlsarah** [comment]
Nice! I like it. Just fix the minor wording and I'll merge it in!

**Comment by lukeberry99** [comment]
Hi @geekygirlsarah I can't see any comments / review for wording changes, can you let me know what you'd like changed and I'll get that done for you asap.

Cheers

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
With Github's review comments, I sometimes forget to hit "submit review" when I'm done. So it was just "pending" this whole time and never sent to you. Sorry about that. 😬

**Comment by lukeberry99** [comment]
@geekygirlsarah no worries! I've also added the 'and' to the title above, as it had the same problem.

File: web/views.py (hunk 1 above)
Comment (geekygirlsarah) [reply web/views.py]: I kind of love that you figured this out before I left the comment, lol

**Comment by geekygirlsarah** [comment]
This is great, thank you for doing this! I'll merge it in.

Don't forget to sign up for Hacktoberfest as this can count towards the shirt/swag!

**Pull request merged**